cypress/integration/User Management/BJ_00_InstructorEdit.js

```diff
@@ -0,0 +1,130 @@
+describe("Fills out form", () => {
+    before(() => {
+        cy.fixture("users.json").then(({instructor, instructor2, inviteInstructorConfirmation}) => {
+            cy.mockGraphQL({
+                "CreateInstructor": {
+                    "response": {
+                        "data": {
+                            "createInstructor": {
+                                "created": false,
+                            },
+                        },
+                    },
+                    "test": (variables) => {
+                        Object.entries(variables).forEach(([key, value]) => {
+                            if (!["phoneNumber", "id", "user", "password", "subjects", "language"].includes(key)) {
+                                expect(instructor2[key] || instructor2?.user[key]).equals(value);
+                            }
+                        });
+                    },
+                },
+                "MyMutation": {
+                    "response": {
+                        "data": {
+                            "inviteInstructor": {
+                                inviteInstructorConfirmation
+                            }
+                        }
+                    },
+                },
+                "GetInstructor": {
+                    "response": {
+                        "data": {
+                            instructor
+                        },
+                    },
+                    "test": ({userID}) => {
+                        expect(userID).equals(instructor.user.id.toString(), "Check ID passed");
+                    },
+                },
+            });
+            cy.visitAuthenticated(`/form/instructor/${instructor.user.id}`);
+        });
+    });
+
+    it("Loads data properly into the first form", () => {
```

Comment:
Be more descriptive of what this is testing for - what form is this testing? what data?